src/components/ObsControl.jsx

```diff
@@ -25,13 +25,17 @@ export class ObsControl extends addMoreDecorator(Component) {
     this.setAbnormal = this.setAbnormal.bind(this);
   }
 
-  onValueChangeDone() {
-    if (this.props.onEventTrigger) {
+  isAllowedToTriggerControlEvent(triggerControlEvent) {
+    return triggerControlEvent === undefined || triggerControlEvent;
```

Comment:
It says `allowToTriggerControlEvent` when `triggerControlEvent` is `undefined`. When it is `undefined`?

Reply:
Other than componentDidMount, every where it is undefined. That means after component mounted successfully and when some change is performed on that component(then we trigger the `onChange` inherited from the parent), we don't pass triggerControlEvent). It then takes ass `undefined`.